tests/jballerina-debugger-integration-test/src/test/resources/testng.xml

```diff
@@ -30,21 +30,30 @@ under the License.
             </run>
         </groups>
         <classes>
-<!--            <class name="org.ballerinalang.debugger.test.remote.BallerinaRunRemoteDebugTest"/>-->
-<!--            <class name="org.ballerinalang.debugger.test.remote.BallerinaTestRemoteDebugTest"/>-->
-<!--            <class name="org.ballerinalang.debugger.test.remote.BallerinaBuildRemoteDebugTest"/>-->
-<!--            <class name="org.ballerinalang.debugger.test.adapter.ModuleBreakpointTest"/>-->
-<!--            <class name="org.ballerinalang.debugger.test.adapter.run.SingleBalFileRunDebugTest"/>-->
-<!--            <class name="org.ballerinalang.debugger.test.adapter.run.MultiModuleRunDebugTest"/>-->
-<!--            <class name="org.ballerinalang.debugger.test.adapter.test.MultiModuleTestDebugTest"/>-->
-<!--            <class name="org.ballerinalang.debugger.test.adapter.build.MultiModuleBuildDebugTest"/>-->
-<!--            <class name="org.ballerinalang.debugger.test.adapter.variables.VariableVisibilityTest"/>-->
-<!--            <class name="org.ballerinalang.debugger.test.adapter.ServiceDebugTest"/>-->
-<!--            <class name="org.ballerinalang.debugger.test.adapter.LanguageConstructDebugTest"/>-->
-<!--            <class name="org.ballerinalang.debugger.test.adapter.ControlFlowDebugTest"/>-->
-<!--            <class name="org.ballerinalang.debugger.test.adapter.evaluation.ExpressionEvaluationTest"/>-->
-<!--            <class name="org.ballerinalang.debugger.test.adapter.evaluation.ExpressionEvaluationNegativeTest"/>-->
-<!--            <class name="org.ballerinalang.debugger.test.adapter.CallStackDebugTest"/>-->
+            <!--Remote Debugging Tests-->
+            <class name="org.ballerinalang.debugger.test.remote.BallerinaRunRemoteDebugTest"/>
+            <class name="org.ballerinalang.debugger.test.remote.BallerinaTestRemoteDebugTest"/>
+            <class name="org.ballerinalang.debugger.test.remote.BallerinaBuildRemoteDebugTest"/>
+
+            <!--Debug engaging tests for ballerina commands -->
+            <class name="org.ballerinalang.debugger.test.adapter.run.SingleBalFileRunDebugTest"/>
+            <class name="org.ballerinalang.debugger.test.adapter.ModuleBreakpointTest"/>
+            <class name="org.ballerinalang.debugger.test.adapter.run.MultiModuleRunDebugTest"/>
+            <!--<class name="org.ballerinalang.debugger.test.adapter.test.MultiModuleTestDebugTest"/>-->
+            <!--<class name="org.ballerinalang.debugger.test.adapter.build.MultiModuleBuildDebugTest"/>-->
```

Comment:
Shall we enable these tests as well? Do we have any issues in that?

Reply:
Please refer #26942.